src.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
+	"folders": [
+		{
+			"path": "..\\..\\Qiagen\\dPCR-software-suit"
+		}
+	]
+}
```

Comment:
please add a new line at the end of the file (see the little red minus)
please format the document (right-click inside your open document in VS Code) > format document